library/core/src/main/java/com/google/android/exoplayer2/trackselection/TrackSelectionParameters.java

```diff
@@ -86,6 +86,14 @@ public Builder() {
      * @return This builder.
      */
     public Builder setPreferredAudioLanguage(@Nullable String preferredAudioLanguage) {
+      if (preferredAudioLanguage == null) {
+        return setPreferredAudioLanguage(new String[0]);
+      } else {
+        return setPreferredAudioLanguage(new String[] { preferredAudioLanguage });
+      }
+    }
+
+    public Builder setPreferredAudioLanguage(String[] preferredAudioLanguage) {
```

Comment:
For the setters it's probably best to use "String..." as type to allow passing in a literal list, like `setPreferredAudioLanguages("FR", "IT")`. If you do this, your own code above can be simplified because you no longer need to create an array.